### DISCUSSION TITLE
Oh 'forge-topic-list-columns' where art thou

### DISCUSSION THREAD

**Discussion by jtamagnan** [magit/forge#679] [Ideas]
# Problem

Before the large topic refactor it was possible to customize the
columns that appeared in the topic list and the global topic list via
`forge-topic-list-columns` and
`forge-global-topic-list-columns`. After looking into
087048f6ff74cc7f82dc1d90f8d4eea2f8e29fb9 I don't see a way to restore
this customization. When I search for `defcustom.*column` all I find
is `forge-repository-list-columns`.

# Request

It would be great if it were once again possible to customize the
columns in the topic lists.

---

I've really been enjoying this new topic view, especially how intuitive
it is to filter and change views within it. Thank you for all the work
that you've done on this package and the entire ecosystem.

Best,
Jules

**Comment by tarsius** [comment]
I'm aware that I dropped some functionality, and as much as I would have liked to restore everything after the refactoring, I had to draw the line somewhere, to avoid infinitely delaying the release Forge (and Magit and half a dozen other packages, which all have to be released at the same time).

I'll probably restore this and similar features, or provide something similar, but at least in some cases that won't happen before the releases.

**Comment by jtamagnan** [comment]
That sounds great. Thanks for the quick reply and incredible ecosystem

Best,
Jules

**Reply by tarsius** [reply]
> I've really been enjoying this new topic view, especially how intuitive it is to filter and change views within it.

Thanks! I needed to hear that.
